bin/utils/process-definition.sh

```diff
@@ -14,10 +14,10 @@ if [[ ! -e ${definition_output_file} ]]; then
    touch ${definition_output_file}
 fi
 
-docker run --rm --name json2xlsx \
+docker run --rm --name json2xlsx --user=$UID\
```

Comment:
take care about that is not for local setup that parameters?

Reply:
oopsie you are right Sergio will revert the change